.ruby-version

```diff
@@ -1 +1 @@
-ruby-2.2.3
+ruby-2.5.0
```

Comment:
The README and `.ruby-version` say 2.5.0, but the PR title and `.travis.yml` say 2.4.3.  Which version are we targeting?